src/app/screens/AddAccount/AddAccountMnemonic.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react';
+
+import { useHistory } from 'react-router-dom';
+
+import { useAccounts } from '@app/hooks';
+import { ROUTE_PATHS } from '@app/routing';
+import { getAddress } from '@app/services';
+import { GetMnemonicAddressesResult, WalletType } from '@types';
+
+const dPathBase = "m/44'/60'/0'/0";
```

Comment:
Is support for other derivation paths planned in the future?

Reply:
Yeah I would say we should do that.